tests/integration/modules/test_tag.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+import saltext.vmware.modules.tag as tagging
+
+
+def test_category_list(patch_salt_globals_tag):
+    """
+    Test create tag functionality
+    """
+    cats = tagging.list_category()
+    if len(cats["categories"]) > 0:
+        assert "urn:vmomi:InventoryServiceCategory" in cats["categories"][0]
+    else:
+        pytest.skip("test requires at least one category")
```

Comment:
**question** should we really skip the test, or can we write a fixture that ensures that a category exists?

Reply:
Note - the fixture should also just return it, and then this test should depend on the fixture and assert equality.